content/blog/2019/09/2019-09-22-google-summer-of-code-mentor-and-org-admin-perspective.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Google Summer of Code Mentor and Org Admin Perspective"
+tags:
+- gsoc
+- gsoc2019
+- jenkins
+- community
+- opensource
+- events
+- mentor
+- orgadmin
+author: markyjackson-taulia
+---
+
+I was fortunate enough to participate in the Google Summer of Code 2019 as a mentor and org admin. This was great
+and I wanted to share in hopes of encouraging more people to join.
+You can learn more about the Google Summer of Code here: https://jenkins.io/projects/gsoc/
+
+== Community Bonding
+
+The first phase of the project is the community bonding phase. This is where the student and other mentors come together
+to lay out the plan for the project. It is important to set expectations and ensure that the student is well aware of
+what will take place and also made to feel welcome.
+
+Parichay Barpanda was the student and he was super awesome from the get go. The project he was working on was the Gitlab
+Branch Source Plugin. More can be found here: https://github.com/jenkinsci/gitlab-branch-source-plugin
```

Comment:
wouldn't it be better to use links like `link:https://github.com/jenkinsci/gitlab-branch-source-plugin[jenkisci/gitlab-branch-source-plugin]`